queue_services/entity-emailer/src/entity_emailer/email_templates/common/business-info.html

```diff
@@ -19,14 +19,12 @@
 
   <tr>
     <td class="name">Incorporation Number:</td>
-    {% if header.status == 'PAID' %}
+    {% if header.status == 'PAID' and filing_type == 'incorporationApplication'
+    %}
```

Comment:
I think there is confusion here. The bug is that the Incorporation Number displays as "Pending"; instead the actual incorporation number should be displayed.

Reply:
It shows "Pending" only when the filing_type is incorporationApplication and status is Paid (at this point we don't have  Incorporation Number). In all other cases (including COD, COA) it shows Incorporation Number. Isn't that the expectation?

Reply:
The description of the bug refers to a Director Change (aka COD) filing confirmation email. In this case the business is already incorporated and has an incorporation number.
I am not sure that this code change is necessary.

Reply:
We should use the term **identifier**, as in our model, taken from json-ld/schema.org
Using too many different terms, will make it less clear.

Reply:
@severinbeauvais at this case the filing_type will be `changeOfDirectors`, so it shows incorporation number.

Reply:
@thorwolpert I am not aware of json-ld/schema.org, can you explain that?

Reply:
Vysakh wrote:
> at this case the filing_type will be changeOfDirectors, so it shows incorporation number

Aaah, OK, now I get it: you're saying that this code inadvertently displayed "Pending" for all filings (including COD), but this code should apply only to IAs.

Reply:
Please create a new conversation for the naming/schema comment, and then resolve THIS conversation.